### DISCUSSION TITLE
Possible to add more panes?

### DISCUSSION THREAD

**Discussion by beolafsen** [oqtane/oqtane.framework#2227] [Q&A] [answered]
Hi!
Would it be possible to have more panes added to the framework? I am thinking in particular of expanding the 'Top Full Width Panel'.
Something like this!
![image](https://user-images.githubusercontent.com/76835718/170976730-d6ea4280-b78f-4924-b76f-2f7c36289a98.png)

Attaches solution if it is of interest:
`        <div class="container-fluid">
            <div class="row">
                <div class="col-md-6">
                    <Pane Name="Full Left 50%" />
                </div>
                <div class="col-md-6">
                    <Pane Name="Full Right 50%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-9">
                    <Pane Name="Full Left 75%" />
                </div>
                <div class="col-md-3">
                    <Pane Name="Full Right 25%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-3">
                    <Pane Name="Full Left 25%" />
                </div>
                <div class="col-md-9">
                    <Pane Name="Full Right 75%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-10">
                    <Pane Name="Full Left 80%" />
                </div>
                <div class="col-md-2">
                    <Pane Name="Full Right 20%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-2">
                    <Pane Name="Full Left 20%" />
                </div>
                <div class="col-md-10">
                    <Pane Name="Full Right 80%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-4">
                    <Pane Name="Full Left 33%" />
                </div>
                <div class="col-md-4">
                    <Pane Name="Full Center 33%" />
                </div>
                <div class="col-md-4">
                    <Pane Name="Full Right 33%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-3">
                    <Pane Name="Full Left Outer 25%" />
                </div>
                <div class="col-md-3">
                    <Pane Name="Full Left Inner 25%" />
                </div>
                <div class="col-md-3">
                    <Pane Name="Full Right Inner 25%" />
                </div>
                <div class="col-md-3">
                    <Pane Name="Full Right Outer 25%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-3">
                    <Pane Name="Full Left 25%" />
                </div>
                <div class="col-md-6">
                    <Pane Name="Full Center 50%" />
                </div>
                <div class="col-md-3">
                    <Pane Name="Full Right 25%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-8">
                    <Pane Name="Full Left Sidebar 66%" />
                </div>
                <div class="col-md-4">
                    <Pane Name="Full Right Sidebar 33%" />
                </div>
            </div>
            <div class="row">
                <div class="col-md-4">
                    <Pane Name="Full Left Sidebar 33%" />
                </div>
                <div class="col-md-8">
                    <Pane Name="Full Right Sidebar 66%" />
                </div>
            </div>
        </div>
`

`    public override string Panes => PaneNames.Admin + ",Top Full Width,Full Left 50%,Full Right 50%,Full Left 80%,Full Right 20%,Full Left 20%,Full Right 80%,Full Left 75%,Full Right 25%,Full Left 25%,Full Right 75%,Full Left 33%,Full Center 33%,Full Right 33%,Full Left Outer 25%,Full Left Inner 25%,Full Right Inner 25%,Full Right Outer 25%,Full Left 25%,Full Center 25%,Full Right 25%,Full Left Sidebar 66%,Full Right Sidebar 33%,Full Left Sidebar 33%,Full Right Sidebar 66%,Top 100%,Left 50%,Right 50%,Left 33%,Center 33%,Right 33%,Left Outer 25%,Left Inner 25%,Right Inner 25%,Right Outer 25%,Left 25%,Center 50%,Right 25%,Left Sidebar 66%,Right Sidebar 33%,Left Sidebar 33%,Right Sidebar 66%,Bottom 100%,Bottom Full Width,Footer";
`

Example: Added 2 status-modules to the left side of the screen (for test). Here I use the 80/20 pane.
![image](https://user-images.githubusercontent.com/76835718/170979097-5faf53ca-5ea9-4b2c-bb90-a418d00aaac0.png)

**Comment by leigh-pointer** [comment, marked as answer]
@beolafsen I have created a theme RocketFuel that allows you to extend the width of the Panes. Overview can been seen here https://studio-elf.net/oqtane/rocketfuel

**Reply by beolafsen** [reply]
Great! I will look into it. 
Thanks.

**Comment by sbwalker** [comment]
@beolafsen the Oqtane theme is only meant to be a functional example. If you need a theme with additional panes or behaviors you should create your own custom theme. Themes are very simple to create - go to the Admin Dashboard, Theme Management, and choose the Create Theme option. It will scaffold a theme which you can modify.

**Reply by beolafsen** [reply]
Thank you - I will look at it.
